Source/Core/DolphinWX/Debugger/CodeWindowFunctions.cpp

```diff
@@ -202,6 +204,22 @@ void CCodeWindow::OnSymbolsMenu(wxCommandEvent& event)
     NotifyMapLoaded();
     break;
   }
+  case IDM_SCAN_RSO:
+  {
+    RSOChainView rso_chain;
```

Comment:
This still applies. This can go within the `if (dialog.ShowModal() == wxID_OK)` case.